low-level/message_handlers/node_data_msg_handler.py

```diff
@@ -605,6 +631,12 @@ def _generate_if_data(self):
 
         nw_alerts = self._get_nwalert(interfaces)
 
+        # Check if other node is reachable on the pvt data nw
+        # Before that, take a copy of prev value
+        other_node = 'srvnode-2' if node_key_id == 'srvnode-1' else 'srvnode-1'
+        self.prev_is_other_node_reachable = self.is_other_node_reachable
+        self.is_other_node_reachable = self._check_ping_to(other_node)

```

Comment:
if other node not reachable (superset of alert), then only we should do check for priv data network port check(subset) & cable status check (subset), to figure out possible reason. or we have option to skip those.